src/main/java/com/google/crypto/tink/aead/subtle/AesGcmSiv.java

```diff
@@ -58,6 +58,18 @@ protected Cipher initialValue() {
   private static final int IV_SIZE_IN_BYTES = 12;
   private static final int TAG_SIZE_IN_BYTES = 16;
 
```

Comment:
I think this is slightly easier to understand if you just do:

private static final boolean HAS_GCM_PARAMETER_SPEC_CLASS = checkForGCMParameterSpec();

private static final boolean checkForGCMParameterSpec() {
    try {
      Class.forName("javax.crypto.spec.GCMParameterSpec");
    } catch (ClassNotFoundException e) {
       return false;
    }
  return true;
}


WDYT?

Reply:
Sorry for the late response.

No objection for this. I will send a new commit!

Reply:
Fixed it, PTAL 🙏.